### DISCUSSION TITLE
Composer metapackage for HttPlugHandle

### DISCUSSION THREAD

**Discussion by freezysko** [phpro/soap-client#360] [Ideas]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | no
| BC Break    | no

#### Summary

Right now, when you wants to use HttPlugHandle, you must install many packages manually with
```
composer require psr/http-message:^1.0 php-http/httplug:^1.1 php-http/message-factory:^1.0 php-http/discovery:^1.3 php-http/message:^1.6 php-http/client-common:^1.6
```

What about making this process a bit easier by creating composer metapackage with all listed packages and their required versions?

```
{
    "name": "phpro/soap-client-httplug",
    "type": "metapackage",
    "require": {
        "php-http/client-common": "^1.6",
        "php-http/discovery": "^1.3",
        "php-http/httplug": "^1.1",
        "php-http/message": "^1.6",
        "php-http/message-factory": "^1.0",
        "psr/http-message": "^1.0"
    }
}
```

**Comment by veewee** [comment]
@janvernieuwe We could use http-tools for this instead and integrate it more with the soap-client?

**Comment by janvernieuwe** [comment]
Sounds like a good idea, once http-tools is a bit more matured.

**Comment by veewee** [comment]
@janvernieuwe  FYI : made http-tools stable:
https://github.com/phpro/http-tools/releases/tag/v1.0.0
